### PR TITLE
[FIX] Fixes unable to see other User profile details

### LINKED ISSUE #2195: I got an error message when I tapped on another users image in the general chat room to have a private conversation.
## Describe the bug
 When I was chatting in the general room, I met some people I wanted to have a privately chat with, so I clicked on their profile images to chat them up, and got an error message saying (''sorry, an error has occurred, pls try again''). The surprising part is that not all does that. just some. I was able to figure out that most names that starts with letter M, L, A . F, K, brings error most times though some works. 

## To Reproduce
1. Go to General chat after login in. (I logged in with my gmail)
2. Click on the profile image of any one you wish to chat with through D.M
3. See error. If the one you clicked works fine, click the back arrow to go back. Try it with different names. You will note the error.

## Expected behavior
After clicking on users image, I expected to be taken to a page where I will be able to click on message or video call to interact with the user in privacy.

## Screenshots

Video recording
https://youtu.be/3d9NVwhmKKw

## Devices and Versions

 Rocket.Chat.Android version: (. 3.3.0) (2060)

Device used - Tecno spark K7 
Version - Android 7.0

## Additional context
At first it looked like the profiles with pictures mostly don't work but, I later noticed some of it works. The problem is almost a general one. 

Note: I can see that the bug in contact us has been fixed. Well done.

### PR DESCRIPTION
**Pull request by Cool-fire**
@RocketChat/android

Closes #2195 #2272 

#### Changes: 
checked if any one of the  `avatarUrl`, `username`, `name`, `utcOffset` is not null. If any of the above is null made an empty String as Default And if utcOffset is null made `None` as default. Added `text_video_call` to  widget group.

### CODE REVIEW

File: app/src/main/java/chat/rocket/android/userdetails/ui/UserDetailsFragment.kt [hunk 1]
```diff
@@ -97,19 +97,23 @@ class UserDetailsFragment : Fragment(), UserDetailsView {
...
-        text_description_timezone.text = utcOffset
+        text_name.text = name ?: ""
+        text_username.text = username ?: ""
 
-        // We should also setup the user details listeners.
-        text_message.setOnClickListener { presenter.createDirectMessage(username) }
+        val userStatus = if(status != null) status.substring(0, 1).toUpperCase() + status.substring(1) else  ""
+        text_description_status.text = userStatus
+
+        val userUtcOffset = if(utcOffset != "null") utcOffset else "None"
```
Comment (philipbrito) [inline app/src/main/java/chat/rocket/android/userdetails/ui/UserDetailsFragment.kt]: Hard-coded string, please remove.

File: app/src/main/java/chat/rocket/android/userdetails/presentation/UserDetailsView.kt [hunk 2]
```diff
@@ -16,11 +16,11 @@ interface UserDetailsView : LoadingView, MessageView {
      * @param isVideoCallAllowed True if the video call is allowed, false otherwise.
      */
     fun showUserDetailsAndActions(
-        avatarUrl: String,
-        name: String,
-        username: String,
-        status: String,
-        utcOffset: String,
+        avatarUrl: String?,
```
Comment (philipbrito) [inline app/src/main/java/chat/rocket/android/userdetails/presentation/UserDetailsView.kt:19]: No need to define it as null on the **interface**.

**Review by philipbrito — CHANGES_REQUESTED** [review]

File: app/src/main/java/chat/rocket/android/userdetails/ui/UserDetailsFragment.kt (hunk 1 above)
Comment (Cool-fire) [reply app/src/main/java/chat/rocket/android/userdetails/ui/UserDetailsFragment.kt]: @filipedelimabrito removed hard coded string and made a new string

**Comment by Cool-fire** [comment]
@filipedelimabrito made the necessary changes, Please review

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -381,6 +381,7 @@ https://github.com/RocketChat/java-code-styles/blob/master/CODING_STYLE.md#strin
     <string name="timezone">Timezone</string>
     <string name="status" translatable="false">Status: %1$s</string>
     <string name="user_detail_status" translatable="false">Status</string>
+    <string name="user_timezone_none" translatable="false">None</string>
```
Comment (philipbrito) [inline app/src/main/res/values/strings.xml]: None is a translatable string.

**Review by philipbrito — CHANGES_REQUESTED** [review]

File: app/src/main/res/values/strings.xml (hunk 3 above)
Comment (Cool-fire) [reply app/src/main/res/values/strings.xml]: @filipedelimabrito added strings in respective languages and added Todo's

**Comment by Cool-fire** [comment]
@filipedelimabrito made the necessary changes. Please review

**Comment by Cool-fire** [comment]
@filipedelimabrito made the necessary changes. Please review

File: app/src/main/java/chat/rocket/android/userdetails/ui/UserDetailsFragment.kt [hunk 4]
```diff
@@ -97,19 +97,23 @@ class UserDetailsFragment : Fragment(), UserDetailsView {
...
-        text_description_timezone.text = utcOffset
+        text_name.text = name ?: ""
+        text_username.text = username ?: ""
 
-        // We should also setup the user details listeners.
-        text_message.setOnClickListener { presenter.createDirectMessage(username) }
+        val userStatus = if(status != null) status.substring(0, 1).toUpperCase() + status.substring(1) else  ""
+        text_description_status.text = userStatus
+
+        val userUtcOffset = if(utcOffset.equals("null")) getString(R.string.user_timezone_none) else utcOffset
```
Comment (philipbrito) [inline app/src/main/java/chat/rocket/android/userdetails/ui/UserDetailsFragment.kt]: Not a good way to check if a variable is null by comparing it's string value.

Comment (philipbrito) [reply app/src/main/java/chat/rocket/android/userdetails/ui/UserDetailsFragment.kt]: Also, there is no reason to create `userUtcOffset `

**Review by philipbrito — APPROVED** [review]
I've improved the code here.
Thanks for sending this PR 👍 .

**Pull request merged**